google/bigtable/admin/v2/bigtable_admin_gapic.yaml

```diff
@@ -0,0 +1,300 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.bigtable.admin.spi.v2
+  python:
+    package_name: google.cloud.gapic.bigtable.admin.v2
+  go:
+    package_name: cloud.google.com/go/bigtable/admin/apiv2
+  csharp:
+    package_name: Google.Bigtable.Admin.V2
+  ruby:
+    package_name: Google::Cloud::Bigtable::Admin::V2
+  php:
+    package_name: Google\Cloud\Bigtable\Admin\V2
+  nodejs:
+    package_name: admin.v2
+license_header:
+  copyright_file: copyright-google.txt
+  license_file: license-header-apache-2.0.txt
+interfaces:
+- name: google.bigtable.admin.v2.BigtableInstanceAdmin
+  collections:
+  - name_pattern: projects/{project}
+    entity_name: project
+  - name_pattern: projects/{project}/instances/{instance}
+    entity_name: instance
+  - name_pattern: projects/{project}/instances/{instance}/clusters/{cluster}
+    entity_name: cluster
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 20000
+    total_timeout_millis: 600000
+  methods:
+  - name: CreateInstance
+    flattening:
+      groups:
+      - parameters:
+        - parent
+        - instance_id
+        - instance
+        - clusters
+    required_fields:
+    - parent
+    - instance_id
+    - instance
+    - clusters
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    field_name_patterns:
+      parent: project
+    timeout_millis: 60000
+  - name: GetInstance
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: instance
+    timeout_millis: 60000
+  - name: ListInstances
```

Comment:
The page streaming config is missing because there are two repeated fields in the response object. I looked at adding the page streaming config, but I am not sure we want to, as it makes accessing the other field more difficult and much less obvious. What do you think?